e2e/e2etests/helpers.go

```diff
@@ -36,3 +37,21 @@ func formatDuration(d time.Duration) string {
 	seconds := d.Seconds() - float64(minutes*60)
 	return fmt.Sprintf("%dm%.1fs", minutes, seconds)
 }
+
+func requireCctxStatus(
```

Comment:
Context on this function? Why not calling require.EqualValues directly in the test? That's how we do for most of the tests right now

Reply:
Answered above , It would be a good idea to start using this function to print cctx details  on failure